packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js

```diff
@@ -1317,7 +1317,9 @@ class VirtualizedList extends React.PureComponent<Props, State> {
   _onCellLayout(e, cellKey, index) {
     const layout = e.nativeEvent.layout;
     const next = {
-      offset: this._selectOffset(layout),
+      // OffsetTop gets the correct offset for each cell for inverted lists beacuse it ignores
+      // parent's transformations (like 'transform: [{scaleY: -1}]').
+      offset: !this.props.horizontal ? layout.offsetTop : layout.x,
```

Comment:
Here we could get `offsetTop` directly from `e.target.offsetTop` rather than wiring it through all the other APIs